statistics/update_test.go

```diff
@@ -721,6 +721,6 @@ func (s *testStatsUpdateSuite) TestUpdateStatsByLocalFeedback(c *C) {
 	c.Assert(tbl.Indices[tblInfo.Indices[0].ID].CMSketch.QueryBytes(low), Equals, uint32(2))
 
 	c.Assert(tbl.Indices[tblInfo.Indices[0].ID].ToString(1), Equals, "index:1 ndv:2\n"+
-		"num: 2\tlower_bound: NULL\tupper_bound: 2\trepeats: 0\n"+
-		"num: 4\tlower_bound: 3\tupper_bound: \trepeats: 0")
+		"num: 2\tlower_bound: \tupper_bound: 2\trepeats: 0\n"+
```

Comment:
lower_bound is displayed as nothing?

Reply:
Because query ` b <= 5` is transformed to `-inf < b <= 5`, after encoding the `-inf`, it is same as an empty string.